src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml

```
@@ -0,0 +1,22 @@
+# analyze-magic
+name: ${{openSystem.content.runeMagic.analyzeMagic.name}}$
+type: runeMagic
+img: systems/rqg/assets/images/items/rune-magic.svg
+data:
+  runes:
+    - Truth (power)
+  points: 1
+  castingRange: touch
+  duration: instant
+  isStackable: true
+  isOneUse: false
+  chance: 0
+  descriptionRqidLink:
+    rqid: je..analyze-magic
+    name: ${{openSystem.content.runeMagic.analyzeMagic.name}}$
+flags:
+  rqg:
+    documentRqidFlags:
+      id: i.runeMagic.rune-magic-analyze-magic
```

Double "rune magic" in id, applies to more rqids as well. 
```suggestion
      id: i.rune-magic.analyze-magic
```

---

Again, I just did file movement so this we missed before. I will review all spells and descriptions. The description links seems to work so it is most likely just the spells (and we don't link to them yet)

---

Cleaned this up and removed links to other spells that will not work.